src/main/java/seedu/address/logic/commands/RemarkCommand.java

```diff
@@ -0,0 +1,101 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_REMARK;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_CANDIDATES;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.candidate.Candidate;
+import seedu.address.model.candidate.Remark;
+
+/**
+ * Changes the remark of an existing candidate in the address book.
+ */
+public class RemarkCommand extends Command {
+
+    public static final String COMMAND_WORD = "remark";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the remark of the candidate identified "
+            + "by the index number used in the last candidate listing. "
+            + "Existing remark will be overwritten by the input.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + PREFIX_REMARK + "[REMARK]\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_REMARK + "Likes to swim.";
```

Comment:
Waiiiiitttt a minute... HAHA

Maybe we could do something related to Interviews? 😝 

EDIT: I think this is a valid remark. Doesn't have to be related to interviews. Like `src/test/java/seedu/address/logic/commands/CommandTestUtil.java`.